src/pages/invites/index.tsx

```diff
@@ -0,0 +1,236 @@
+                      value={editingInviteName}
+                      onChange={(e) => setEditingInviteName(e.target.value)}
+                      autoFocus
+                    />
+                    <button
+                      className="flex-item mb-3"
+                      onClick={() => handleSaveClick()}
+                    >
+                      <img src={SaveImage.src} alt="Save changes" width={32} />
+                    </button>
+                  </div>
+                ) : (
+                  <div className="flex items-center edit-container">
+                    <p className="text-sm font-medium text-gray-900 truncate dark:text-white flex-item">
+                      {invite.name || invite.id}
+                    </p>
+                    <img
+                      className={`flex-item cursor-pointer ${
+                        hoverInviteId === invite.id ? '' : 'hidden'
+                      }`}
+                      width={15}
+                      src={EditImage.src}
+                      alt="Edit the name"
+                      onClick={() => handleEditClick(invite)}
+                    />
+                    <img
+                      className={`flex-item cursor-pointer ${
+                        hoverInviteId === invite.id ? '' : 'hidden'
+                      }`}
+                      width={15}
+                      src={DeleteImage.src}
+                      alt="Delete the invite"
+                      onClick={() => handleDeleteClick(invite)}
+                    />
+                    <img
+                      className={`flex-item cursor-pointer ${
+                        hoverInviteId === invite.id ? '' : 'hidden'
+                      }`}
+                      width={18}
+                      src={DollarImage.src}
+                      alt="Top Up the invite"
+                      onClick={() => onTopUpInvite && onTopUpInvite(invite)}
+                    />
+                  </div>
+                )}
+                <div
+                  className="flex items-center text-sm text-gray-400"
+                  onClick={() => {
+                    copy(makeInviteUrl(invite.invite));
+                    alert('Invite copied to clipboard');
+                  }}
+                >
+                  <div>
+                    <img
+                      className="flex-item cursor-pointer"
+                      src={Key.src}
+                      alt="Click to copy the invite"
+                      width={15}
+                    />
+                  </div>
+                  <div
+                    className="flex-item truncate blurred-text cursor-pointer"
+                    title="Click to copy the invite"
+                  >
+                    0x5416f2b3c072a29e577b6fcfa2230671958aaa4c759bb2e37d4bd647632aa76c
```

Comment:
Should this be hard-coded?

Reply:
yes. it's just needed for UI blur effect. Real data is not suitable because of security reasons (blur can be reverted).